src/core/Tooltip/TooltipContent/TooltipContent.baseStyles.tsx

```diff
@@ -20,20 +20,21 @@ export const baseStyles = (arrowOffsetPx: number, theme: SuomifiTheme) => css`
       position: absolute;
       height: 0;
       width: 0;
-      bottom: 100%;
       left: ${arrowOffsetPx}px;
       border: solid transparent;
       pointer-events: none;
     }
     &:before {
       border-bottom-color: ${theme.colors.depthDark2};
```

Comment:
Would it make sense to use a bit darker color here? The line color looks lighter when it's in an angle, compared to the box borders.  For example depthDark1 is a bit darker.

Reply:
Looks better now. On macbook screen you can notice that it's darker than other borders. Then in the external HP screen it's a bit lighter than others. I suppose in average it's good 😄

Reply:
This seems interesting problem; when using same colour, `depthDark2` it actually looks a lot lighter as mentioned. But when using the `blackBase` it looks nice until zooming closer.

So as a compromise I would vote also for `blackBase` for it to look visually pleasant in normal use. We can change it later if this is a real issue or we can figure some other solution for it.